cmd/storage-rest-client.go

```diff
@@ -291,32 +291,17 @@ func (client *storageRESTClient) DiskInfo(ctx context.Context, metrics bool) (in
 
 // MakeVolBulk - create multiple volumes in a bulk operation.
 func (client *storageRESTClient) MakeVolBulk(ctx context.Context, volumes ...string) (err error) {
-	values := make(url.Values)
-	values.Set(storageRESTVolumes, strings.Join(volumes, ","))
-	respBody, err := client.call(ctx, storageRESTMethodMakeVolBulk, values, nil, -1)
-	defer xhttp.DrainBody(respBody)
-	return err
+	return errInvalidArgument
```

Comment:
So we keep these as local-only?

Reply:
Maybe we should move these from the `StorageAPI` and have a `type LocalStorage interface {` that we can check against before using them..

Reply:
Yeah thought of it, but its a lot of refactor - let me see if I can do it.

Reply:
actually that looks like a lot of refactor that PR was intended for